### PR TITLE
Update README.me with support of OCP 4.6

### LINKED ISSUE #140: Update documentation on Conjur OSS OpenShift support
AC:
- [ ] Update READMEs to remove "Community" caveats from OCP support
- [ ] Update supported OCP versions to 4.6

### PR DESCRIPTION
**Pull request by rpothier**
### What does this PR do?
Update the README.md in conjur_oss to support OCP 4.6 

### What ticket does this PR close?
Resolves #140 

#### Change log
- [x] The CHANGELOG has been updated, or
- [ ] This PR does not include user-facing changes and doesn't require a CHANGELOG update

#### Test coverage
- [ ] This PR includes new unit and integration tests to go with the code changes, or
- [x] The changes in this PR do not require tests

#### Documentation
- [ ] Docs (e.g. `README`s) were updated in this PR, and/or there is a follow-on issue to update docs, or
- [x] This PR does not require updating any documentation

### CODE REVIEW

**Review by diverdane — APPROVED** [review]
LGTM.  Although we may want to wait for @doodlesbykumbi testing to be merged for the kubernetes-conjur-demo?

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## Unreleased
 
+### Added
+- Support for deploying Conjur OSS to OpenShift 4.6
```
Comment (izgeri) [inline CHANGELOG.md:10]:
```suggestion
- Support for deploying Conjur OSS to OpenShift 4.6.
  [cyberark/conjur-oss-helm-chart#140](https://github.com/cyberark/conjur-oss-helm-chart/issues/140)
```

**Review by izgeri — APPROVED** [review]
LGTM

**Review by izgeri — APPROVED** [review]
LGTM! Let's squash merge the changes into a single commit once the pipeline is green

**Pull request merged**